spl/token/_layouts.py

```diff
@@ -27,6 +27,7 @@ class InstructionType(IntEnum):
     APPROVE2 = 13
     MINT_TO2 = 14
     BURN2 = 15
+    CREATE_ASSOCIATED_TOKEN_ACCOUNT = 16
```

Comment:
I think this should actually be `0`? B/c the ATA program is a separate program from the SPL token program. And there is no instruction on the 16th index for the ATA program.

Reply:
`INITIALIZE_MINT` is an entirely different program right? `CREATE_ASSOCIATED_TOKEN_ACCOUNT` is not on the spl token program.

Reply:
So we probably want to remove `CREATE_ASSOCIATED_TOKEN_ACCOUNT` from here too:
https://github.com/michaelhly/solana-py/pull/56/files#diff-7cedf59489794c8d47d7bc9f6852d5178b006f239bd3a18caf545df76b86376bR72

Reply:
Basically, we have two programs. And we're pretending they're the same program when they're not.